layers/stateless/sl_render_pass.cpp

```diff
@@ -671,6 +671,22 @@ bool Device::manual_PreCallValidateCmdBeginRendering(VkCommandBuffer commandBuff
     return skip;
 }
 
+bool Device::ValidateRenderingAttachmentInfoPNext(VkCommandBuffer commandBuffer, const void *pNext,
```

Comment:
would just make this 
```suggestion
bool Device::ValidateRenderingAttachmentInfoPNext(VkCommandBuffer commandBuffer, const VkRenderingAttachmentInfo& attachment,
```